### PR TITLE
Cleanup many docstrings.

### PR DESCRIPTION
**Pull request by anntzer**
## PR Checklist

- [ ] Has Pytest style unit tests
- [ ] Code is [Flake 8](http://flake8.pycqa.org/en/latest/) compliant
- [ ] New features are documented, with examples if plot related
- [ ] Documentation is sphinx and numpydoc compliant
- [ ] Added an entry to doc/users/next_whats_new/ if major new feature (follow instructions in README.rst there)
- [ ] Documented in doc/api/api_changes.rst if API changed in a backward-incompatible way

### CODE REVIEW

File: lib/matplotlib/axes/_base.py [hunk 1]
```diff
@@ -1286,16 +1286,16 @@ def set_aspect(self, aspect, adjustable=None, anchor=None, share=False):
...
-        Returns the adjustable parameter, *{'box', 'datalim'}* that defines
-        which parameter the Axes will change to achieve a given aspect.
+        Return whether the Axes will adjust its physical dimension ('box') or
+        its data limits ('datalim') to achieve the desired aspect.
 
         See Also
         --------
         matplotlib.axes.Axes.set_adjustable
-            defining the parameter to adjust in order to meet the required
+            Set the parameter to adjust in order to achieve the required
```
Comment (timhoffm) [inline lib/matplotlib/axes/_base.py]: That sounds complicated
```suggestion
            Set how the Axes adjusts to the required aspect ratio.
```

File: lib/matplotlib/tri/triinterpolate.py [hunk 2]
```diff
@@ -241,13 +241,13 @@ class LinearTriInterpolator(TriInterpolator):
 
     Parameters
     ----------
-    triangulation : `~matplotlib.tri.Triangulation`
+    triangulation : `.Triangulation`
```
Comment (timhoffm) [inline lib/matplotlib/tri/triinterpolate.py]: I think we had that topic in another PR lately. This change is against our documentation guide https://matplotlib.org/devel/documenting_mpl.html#referencing-types
> Use full references `~matplotlib.colors.Normalize` with an abbreviation tilde in parameter types. While the full name helps the reader of plain text docstrings, the HTML does not need to show the full name as it links to it. Hence, the ~-shortening keeps it more readable.
>
> Use abbreviated links `.Normalize` in the text.

I still think this is a reasonable policy, but we can discuss that if you like. However, I don't want us do act differently from our stated policy.

File: lib/matplotlib/tri/triinterpolate.py [hunk 3]
```diff
@@ -304,25 +304,25 @@ class CubicTriInterpolator(TriInterpolator):
...
```
Comment (timhoffm) [inline lib/matplotlib/tri/triinterpolate.py]: see doc policy, as above

File: lib/matplotlib/tri/trirefine.py [hunk 4]
```diff
@@ -51,8 +50,8 @@ class UniformTriRefiner(TriRefiner):
 
     Parameters
     ----------
-    triangulation : :class:`~matplotlib.tri.Triangulation`
-                     The encapsulated triangulation (to be refined)
+    triangulation : `.Triangulation`
```
Comment (timhoffm) [inline lib/matplotlib/tri/trirefine.py]:
```suggestion
    triangulation : `~matplotlib.tri.Triangulation`
```
see doc policy, as above

File: lib/matplotlib/tri/trirefine.py [hunk 5]
```diff
@@ -64,30 +63,29 @@ def __init__(self, triangulation):
...
-            Each triangle will be divided into ``4**subdiv`` child triangles.
+            Each triangle is divided into ``4**subdiv`` child triangles;
+            hence, the default results in 64 refined subtriangles for each
+            triangle of the initial triangulation.
 
         Returns
         -------
-        refi_triangulation : :class:`~matplotlib.tri.Triangulation`
-            The returned refined triangulation
+        refi_triangulation : `.Triangulation`
```
Comment (timhoffm) [inline lib/matplotlib/tri/trirefine.py]:
```suggestion
        refi_triangulation : `~matplotlib.tri.Triangulation`
```
see doc policy, as above

File: lib/matplotlib/tri/trirefine.py [hunk 6]
```diff
@@ -134,31 +132,27 @@ def refine_triangulation(self, return_tri_index=False, subdiv=3):
...
             Recursion level for the subdivision.
-            Each triangle will be divided into ``4**subdiv`` child triangles.
+            Each triangle is divided into ``4**subdiv`` child triangles.
 
         Returns
         -------
-        refi_tri : `~matplotlib.tri.Triangulation`
-                     The returned refined triangulation
+        refi_tri : `.Triangulation`
+             The returned refined triangulation
```
Comment (timhoffm) [inline lib/matplotlib/tri/trirefine.py]:
```suggestion
             The returned refined triangulation.
```

File: lib/matplotlib/tri/trirefine.py [hunk 7]
```diff
@@ -134,31 +132,27 @@ def refine_triangulation(self, return_tri_index=False, subdiv=3):
...
         refi_z : 1d array of length: *refi_tri* node count.
-                   The returned interpolated field (at *refi_tri* nodes)
+             The returned interpolated field (at *refi_tri* nodes)
```
Comment (timhoffm) [inline lib/matplotlib/tri/trirefine.py]:
```suggestion
             The returned interpolated field (at *refi_tri* nodes).
```

File: lib/matplotlib/tri/tritools.py [hunk 8]
```diff
@@ -12,18 +12,19 @@ class TriAnalyzer:
...
+    tools for mesh analysis and mesh improvement.
 
     Attributes
     ----------
     scale_factors
 
     Parameters
     ----------
-    triangulation : `~matplotlib.tri.Triangulation`
+    triangulation : `.Triangulation`
```
Comment (timhoffm) [inline lib/matplotlib/tri/tritools.py]:
```suggestion
    triangulation : `~matplotlib.tri.Triangulation`
```
see doc policy, as above

File: lib/matplotlib/transforms.py [hunk 9]
```diff
@@ -512,8 +511,8 @@ def anchored(self, c, container=None):
               - etc.
 
         container : Bbox, optional
```
Comment (timhoffm) [inline lib/matplotlib/transforms.py]:
```suggestion
        container : `Bbox`, optional
```

File: lib/matplotlib/transforms.py [hunk 10]
```diff
@@ -1551,11 +1537,11 @@ class TransformWrapper(Transform):
     run time with a transform of a different type.  This class allows
     that replacement to correctly trigger invalidation.
 
-    Note that :class:`TransformWrapper` instances must have the same
-    input and output dimensions during their entire lifetime, so the
-    child transform may only be replaced with another child transform
-    of the same dimensions.
+    Note that `TransformWrapper` instances must have the same input and output
```
Comment (timhoffm) [inline lib/matplotlib/transforms.py]:
```suggestion
    `TransformWrapper` instances must have the same input and output
```

File: lib/matplotlib/transforms.py [hunk 11]
```diff
@@ -2780,7 +2769,7 @@ def interval_contains_open(interval, val):
     Returns
     -------
     bool
-        Returns true if given val is within the interval.
+        Whether if given val is within the interval.
```
Comment (timhoffm) [inline lib/matplotlib/transforms.py]:
```suggestion
        Whether *val* is within the *interval*.
```

File: lib/matplotlib/transforms.py [hunk 12]
```diff
@@ -2794,7 +2783,7 @@ def offset_copy(trans, fig=None, x=0.0, y=0.0, units='inches'):
     ----------
     trans : `Transform` subclass
         Any transform, to which offset will be applied.
-    fig : :class:`~matplotlib.figure.Figure`, default: None
+    fig : `.Figure`, default: None
```
Comment (timhoffm) [inline lib/matplotlib/transforms.py]:
```suggestion
    fig : `~matplotlib.figure.Figure`, default: None
```
see doc policy

File: lib/matplotlib/axes/_base.py (hunk 1 above)
Comment (anntzer) [reply lib/matplotlib/axes/_base.py]: better, changed

File: lib/matplotlib/tri/triinterpolate.py (hunk 2 above)
Comment (anntzer) [reply lib/matplotlib/tri/triinterpolate.py]: let's discuss this separately, I reverted the changes for now.

File: lib/matplotlib/tri/triinterpolate.py (hunk 3 above)
Comment (anntzer) [reply lib/matplotlib/tri/triinterpolate.py]: idem

File: lib/matplotlib/tri/trirefine.py (hunk 6 above)
Comment (anntzer) [reply lib/matplotlib/tri/trirefine.py]: yup

**Comment by anntzer** [comment]
thanks, all handled.

**Pull request merged**